src/client/js/partials/floatingBanner.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/** @type {HTMLDivElement} */
+// @ts-ignore: We guard against a null value by not calling init():
+const floatingBannerElement = document.querySelector('.floating-banner')
+
+if (floatingBannerElement) {
+  init()
+}
+
+function init () {
+  const { type, delay } = floatingBannerElement.dataset
+  const userInteractedWithBanner = localStorage.getItem(type) === 'shown'
+  if (!type || !delay || userInteractedWithBanner) {
+    return
+  }
+
+  setTimeout(() => {
+    floatingBannerElement.removeAttribute('hidden')
+    floatingBannerElement.focus()
```

Comment:
Setting the focus on the banner as soon as it appears.

Reply:
Hmm... Won't that get annoying, e.g. if you were just typing something somewhere?